Dangerfile

```diff
@@ -10,7 +10,16 @@ changed_files = git.diff.stats[:files]
 
 excluded_changed_files = changed_files.select { |key| EXCLUSIONS.any? { |exclusion| key.include?(exclusion) } }
 filtered_changed_files = changed_files.reject { |key| EXCLUSIONS.any? { |exclusion| key.include?(exclusion) } }
-lines_of_code = filtered_changed_files.sum { |_file, changes| (changes[:insertions] + changes[:deletions]) }
+
+# ignores whitespace for the purpose of determining lines of code changed
+changes = `git diff -w --stat`.split("\n")
```

Comment:
i think the message output by dangerbot below should also mention it's not counting whitespace changes.